galaxy.yml

```diff
@@ -21,4 +21,4 @@ repository: https://github.com/ansible-collections/ansible.network
 tags: [networking, security, cloud, network_cli, netconf, httpapi, grpc]
 # NOTE(pabelanger): We create an empty version key to keep ansible-galaxy
 # happy. We dynamically inject version info based on git information.
-version: null
+version: 2.0.0
```

Comment:
This won't be 2.0.0 v2.0.0 has already been released and would result in conflicts if someone tries to install it from source. It should be 2.0.1-dev.